Modules/ITS/itsDecoding.json

```diff
@@ -28,14 +28,14 @@
                 "className": "o2::quality_control_modules::its::ITSDecodingErrorTask",
                 "moduleName": "QcITS",
                 "detectorName": "ITS",
-                "cycleDurationSeconds": "30",
+                "cycleDurationSeconds": "10",
```

Comment:
Minor comment: if this is planned also for P2 json, one just need to check whether 10s is not too short. Thinking about possible backpressure issues.